### PR TITLE
External Personnel: Remove functionality due to External Personnel is replaced by Contract Personnel

### PR DESCRIPTION
**Pull request by terjebra**
- [x] New feature
- [ ] ~~Bug fix~~
- [x] High impact

**Description of work:**
Contract Personnel replaces External personnel . This PR removes External personnel functionality from resources. Endpoints are kept and returns 410

[AB#28840]

**Testing:**
- [x] Can be tested
- [x] Automatic tests created / updated
- [x] Local tests are passing

**Checklist:**
- [x] Considered automated tests
- [x] Considered updating specification / documentation
- [x] Considered work items 
- [x] Considered security
- [x] Performed developer testing
- [x] Checklist finalized / ready for review

### CODE REVIEW

File: src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs [hunk 1]
```diff
@@ -158,52 +158,12 @@ public static IAuthorizationRequirementRule CurrentUserIs(this IAuthorizationReq
...
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule BeContractorInContract(this IAuthorizationRequirementRule builder, Guid orgContractId)
-        {
-            builder.AddRule(new ContractorInContractRequirement(orgContractId));
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule ProjectAccess(this IAuthorizationRequirementRule builder, ProjectAccess level, ProjectIdentifier project)
```
Comment (xssic) [inline src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs:173]: Is this related to contracts? (Should it be removed ?)

File: src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs [hunk 2]
```diff
@@ -158,52 +158,12 @@ public static IAuthorizationRequirementRule CurrentUserIs(this IAuthorizationReq
...
-
-        public static IAuthorizationRequirementRule DelegatedContractAccess(this IAuthorizationRequirementRule builder, DelegatedContractRole role, ProjectIdentifier project, Guid contractOrgId)
-        {
-            var resource = new ContractResource(project, contractOrgId);
-            builder.AddRule(resource, role);
-
-            return builder;
-        }
-
-        public static IAuthorizationRequirementRule RequestAccess(this IAuthorizationRequirementRule builder, RequestAccess level, QueryPersonnelRequest request)
```
Comment (xssic) [inline src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs:200]: Is this related to contracts? (Should it be removed ?)

**Review by xssic — COMMENTED** [review]
No need keeping database entities not in use anymore.
Should be removed.

File: src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs (hunk 1 above)
Comment (terjebra) [reply src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs:173]: I did not find any references in the code so I removed it

File: src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs (hunk 2 above)
Comment (terjebra) [reply src/backend/api/Fusion.Resources.Api/Authorization/Extensions/RequirementsBuilderExtensions.cs:200]: Did not find any refereneces so I just removed it

**Review by xssic — APPROVED** [review]

**Comment by xssic** [comment]
Should create a story removing related DB entities.
Add link before resolving
